arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -2198,6 +2184,30 @@ ExecutionBlockImpl<Executor>::executeWithoutTrace(AqlCallStack stack) {
           }
           break;
         }
+        case ExecState::NEXTSUBQUERY: {
```

Comment:
I think we should make clearer that `NEXTSUBQUERY` means the next execution of this subquery.